statistics-production/scrums.qmd

```diff
@@ -41,16 +41,12 @@ If you are responsible for signing off publications, then please download and se
 
 ## Scrum information
 
-You can experience how a scrum runs through watching this scrum-along (features the scrum up until groups breakout to discuss different elements) and [supporting slides](https://educationgovuk.sharepoint.com/:p:/r/sites/lvewp00086/WorkplaceDocuments/CSSU/Statistics%20Strategy%20Team/Content%20Design%20Scrums%20materials/Content%20Design%20Scrum.pptx?d=wb13c1dc2628647718ef9fd1690653547&csf=1&web=1&e=qKLAYn)
+You can find information on scrum along wiht a useful diagram in [Microsoft DevOps](https://learn.microsoft.com/en-us/devops/plan/what-is-scrum) and can experience how a scrum runs through reading the [supporting slides](https://educationgovuk.sharepoint.com/:p:/r/sites/lvewp00086/WorkplaceDocuments/CSSU/Statistics%20Strategy%20Team/Content%20Design%20Scrums%20materials/Content%20Design%20Scrum.pptx?d=wb13c1dc2628647718ef9fd1690653547&csf=1&web=1&e=qKLAYn) to a past scrum along.
 
-<div align=center>
-<iframe width="640" height="360" src="https://web.microsoftstream.com/embed/video/73fa0d31-b8b3-4889-bfcf-c76c33c1fe87?autoplay=false&showinfo=false" allowfullscreen style="border:none;"></iframe>
-</div>
+A full before / after scrum along is available to [watch](https://educationgovuk-my.sharepoint.com/:v:/r/personal/heather_brown_education_gov_uk/Documents/Recordings/Content%20Design%20Scrum_%20Progression%20Measures%20Statistics-20210628_123245-Meeting%20Recording.mp4?csf=1&web=1&e=xc032a), with slides available to [download separately](../resources/Content_Design_Scrum_Progression.pptx), upon request.
 
-A full before / after scrum along is available to [watch](https://educationgovuk-my.sharepoint.com/:v:/r/personal/heather_brown_education_gov_uk/Documents/Recordings/Content%20Design%20Scrum_%20Progression%20Measures%20Statistics-20210628_123245-Meeting%20Recording.mp4?csf=1&web=1&e=xc032a), with slides available to [download separately](../resources/Content_Design_Scrum_Progression.pptx).
+For a walk through of some of the end to end benefits the scrums have had, you may ask for access to this [video for Schools and pupils Statistics Team](https://educationgovuk-my.sharepoint.com/:v:/r/personal/heather_brown_education_gov_uk/Documents/Recordings/Content%20Design%20Scrum_%20Progression%20Measures%20Statistics-20210628_123245-Meeting%20Recording.mp4?csf=1&web=1&e=xc032a).
```

Comment:
I think Heather's left the dept, so don't think any requests are going to get responded to. Think we just need to remove the link & paragraph entirely.